### DISCUSSION TITLE
Smart WiFi3.0 not listed in your app

### DISCUSSION THREAD

**Discussion by itcschmidt** [nathanmarlor/foxess_modbus#278] [Support]
Hi,

I am from Germany and using the FOX ESS Wifi Stick 3.0 with my H3 hybrid inverter.
How to connect the wifi stick for using home assist?

Here teh doc to the stick: https://www.fox-ess.com/download/upfiles/10-203-00113-01%20WiFi3.0%20English%20Installation%20guide.pdf

Thanks
Chris

**Comment by canton7** [comment]
You can't I'm afraid. That data logger connects the inverter to foxcloud, but we can't use it to talk to the inverter ourselves.

You need to connect to your inverter either by connecting a LAN cable to it, [see here](https://github.com/nathanmarlor/foxess_modbus/wiki/Direct-Ethernet-Connection-to-Inverter), or by wiring into its RS485 port, [see here](https://github.com/nathanmarlor/foxess_modbus/wiki/Modbus-Wiring-Guide).

**Comment by Kasttolo** [comment]
Hi, I am having the same issue with a WIFi dongle blocking my Ethernet port. My RS485 port already has a connection to the power meter and I am a bit hesitant to disassemble the plug and add 2 more wires. Do you have a recommendation, what to do?

**Reply by nathanmarlor** [reply]
The recommendation is to connect to the 485 interface for the most advanced features.

**Reply by canton7** [reply]
The WiFi dongle doesn't normally block the ethernet port (although not all inverters have an ethernet port).

**Reply by dreampi** [reply]
I was quite hesitant about using the RS485 method, especially since I have no electrical skills whatsoever. However, when I stumbled upon this integration and read the posted guide, it made me realize that the task isn't as intimidating as I initially thought. In the end, I was able to complete it within 30 minutes without encountering any issues. I highly recommend connecting this way, as it provides full control over your system.

**Reply by Kasttolo** [reply]
Thanks for your response. @dreampi do you also have a smart meter connected to your RS485. My worry is to disassemble the existing plug and mess up the installation

**Reply by dreampi** [reply]
Not exactly, I have a CT clamp connected to my RS485. I had the same concern about potentially messing up the installation. However, as long as you are careful and avoid tugging on the cable during the disassembly process, everything should go smoothly.

**Reply by Kasttolo** [reply]
I have tried it today and it works. You were right, it wasn't that difficult. Many thanks for your help.

**Reply by canton7** [reply]
Nice one!
